client/dashboard/app/dev-tools/preferences.scss

```
@@ -1,15 +1,29 @@
 @import '@wordpress/base-styles/variables';
 
-.preferences-helper__current-preferences {
+.preferences-helper__preference {
 	font-size: $font-size-small;
 	text-transform: none;
+	font-weight: normal;
 }
 
-.environment.is-prefs:not(:hover) .preferences-helper__current-preferences {
+.preferences-helper__preference-content {
+	padding-inline-start: 36px;
+
+	ul {
+		margin: 0;
+		padding: 0;
+
+		li {
+			margin-bottom: 8px;
```

This one is equivalent.

```suggestion
			margin-bottom: $grid-unit-10;
```